bin/svg-sprite.js

```diff
@@ -276,7 +276,17 @@ argv._.reduce((f, g) => [...f, ...glob.sync(g)], [])
             basename = basepos >= 0 ? basename.substr(basepos + 2) : path.basename(file);
         }
 
-        spriter.add(file, basename, fs.readFileSync(file));
+        let openedFile = null;
+
+        try {
+            openedFile = fs.readFileSync(file);
+        } catch (error) {
+            console.error(`[ERROR] Skipping (${file}) due to errors (${error.message.trim()})`);
```

Comment:
Wouldn't it be better to throw here?

It's hard working without tests for bin, but I assume you tested it anyway :)

Reply:
I think you're right. It's better to throw. But github suggest using file descriptors instead. 
I think we can dismiss this alert

Reply:
Sounds good to me. We can revisit it later if needed.